spec/rubocop/cop/style/redundant_parentheses_spec.rb

```diff
@@ -697,6 +697,20 @@ def x
     expect_no_offenses('(a...b)')
   end
 
+  it 'accepts parentheses around an irange when a different expression precedes it' do
+    expect_no_offenses(<<~RUBY)
+      do_something
+      (a..b)
```

Comment:
Hmm, why are the parens needed here?

Reply:
Yeah, the parentheses are not required for grammatical correctness. However, even in the current behavior, parentheses are preserved to respect cases where range literals are explicitly enclosed for clarity.

Reply:
I see. That's not documented anywhere, though, so at the very least it should be made explicit, and even better - configurable.